geometry_msgs/Pose location
long_term_deployment/Task task
time start_after
time finish_before
duration estimated_duration
